app/Models/Maker.php

```diff
@@ -41,7 +41,7 @@ class Maker extends Model
 
     protected $guarded = [];
 
-    public function game(){
+    public function game() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->